infra/terraform/test-org/org/locals.tf

```diff
@@ -875,6 +875,16 @@ locals {
       lint_env = {
         ENABLE_BPMETADATA = "1"
       }
+    },
+    {
+      name            = "kms-solutions"
+      org             = "GoogleCloudPlatform"
+      description     = "Store Cloud KMS scripts, artifacts, code samples, and more."
+      owners          = ["tdbhacks"]
+      enable_periodic = true
```

Comment:
Do we need periodics since not JSS?

Reply:
@romanini-ciandt?

That said, I don't think it should cause any issues.

Reply:
Nop, I don't think we need. It's removed!

Reply:
Thanks @romanini-ciandt - Can you please do a `terraform fmt` on the file.

Reply:
I took care of it.

Reply:
Thank you @apeabody